### PR TITLE
Added note regarding Stream incompatibility with ServiceControl

### PR DESCRIPTION
**Pull request by ramonsmits**
@Particular/azure-maintainers Please review

### CODE REVIEW

**Comment by ramonsmits** [comment]
@Particular/azure-maintainers How can we recommend something that isn't compatible with the rest of our platform?

**Comment by yvesgoeleven** [comment]
@ramonsmits as discussed on slack, because we have to, MS is recommending it going that direction only in the future

File: nservicebus/azure-service-bus/topologies/index.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ The `EndpointOrientedTopology` is backward compatible with the Azure Service Bus
 
 The `ForwardingTopology` was introduced to take advantage of the broker nature of the Azure Service Bus and to leverage its native capabilities. It is the recommended option for new projects.
 
+NOTE: Although the ForwardingTopology is recommended it is currently not compatible with ServiceControl.
```
Comment (SeanFeldman) [inline nservicebus/azure-service-bus/topologies/index.md]: "Although" is not needed.

Comment (SeanFeldman) [inline nservicebus/azure-service-bus/topologies/index.md]: Back tick ForwardingTopology name.

**Comment by ramonsmits** [comment]
@SeanFeldman I've updated the notes.

**Comment by SeanFeldman** [comment]
I'm on a fence with this change.

> currently NOT compatible with ServiceControl.

As a new reader I'll understand it as "no SC for you". While it's not compatible as-is, there's an option of an adapter. And if we warn about incompatibility, we should mention adapter as an option.

@yvesgoeleven @ramonsmits

**Comment by yvesgoeleven** [comment]
@SeanFeldman maybe add a link to the adapter everywhere as well?

PS: I really really feel we should fix the SC, been repeating this for 3 years now, I know, but still...
PPS: No one reads doco until it's too late

**Comment by SeanFeldman** [comment]
> No one reads doco until it's too late

Even then I would not want to read the grim note w/o a ray of hope that **is** possible.

**Comment by yvesgoeleven** [comment]
@SeanFeldman hence the link instead

**Comment by ramonsmits** [comment]
To have a bit of both:

> The `SupportedBrokeredMessageBodyTypes.Stream` body format is recommended, but this mode is currently NOT compatible with ServiceControl Out of the box. As an alternative, it is possible to use the ServiceControl adapter approach to make ServiceControl compatible.

**Pull request merged**